tests/test_twilio_backend.py

```diff
@@ -47,5 +47,17 @@ def test_twilio_sandbox_backend(client, mocker):
 
     mock_twilio_messages.create.assert_called_once_with(to=phone_number, body=message, from_=from_number)
 
+    url = reverse("phone-verify")
```

Comment:
maybe we can break this code at both places in separate tests.

but this is also something we've to take care of while refactoring tests using parameterize factories as we discussed.

Reply:
Yes, I was working on that. I have written them here in the same format as they were already written.

Reply:
Sure, sounds good! We can do this later.